platforms/documentation/docs/src/samples/writing-tasks/custom-test-task/tests/test.sample.conf

```diff
@@ -0,0 +1,12 @@
+commands: [{
+    executable: gradle
+    args: -q test --fail
```

Comment:
💭 It might be easier to follow the example if you split the custom test task into 2 custom tasks - a passing test task and a failing test.  Asking the test to fail via an arg is an extra step for readers to understand.